### PR TITLE
UHF-3322: Updated site configuration based on latest code

### PR DESCRIPTION
**Pull request by teroelonen**
How to test:

1. Checkout this branch on this instance. If you don't have the instance set up yet you need to install the site with this branch `make new`
2. After install or if you already have the site installed run `make drush-cim && make drush-cr`
3. Now go to add content and make sure that the accordion for example has title and description fields or check the footer that there is the correct block "Helsingin kaupunki". Also make sure that the cookie page has all categories and translations.

### CODE REVIEW

File: conf/cmi/language/fi/field.field.paragraph.banner.field_banner_link.yml [hunk 1]
```diff
@@ -1,2 +1 @@
 label: Linkki
-description: 'You can add link button to the banner using this field.'
```
Comment (khalima) [inline conf/cmi/language/fi/field.field.paragraph.banner.field_banner_link.yml:2]: Most of the field description translations are being deleted. Was the translations imported correctly during installation? Try to run `drush locale:check && drush locale:update`.

**Review by khalima — CHANGES_REQUESTED** [review]
Let's discuss about these items.

File: conf/cmi/language/fi/field.field.paragraph.banner.field_banner_link.yml (hunk 1 above)
Comment (teroelonen) [reply conf/cmi/language/fi/field.field.paragraph.banner.field_banner_link.yml:2]: This worked so I was able to add the Finnish texts there now <3

**Review by khalima — APPROVED** [review]
AFAIK This is good to go now! 🚀

**Pull request merged**